modules/reindex/src/main/java/org/elasticsearch/index/reindex/AbstractAsyncBulkByScrollAction.java

```diff
@@ -275,23 +285,27 @@ void onBulkResponse(BulkResponse response) {
                 startNormalTermination(emptyList(), emptyList());
                 return;
             }
-            startNextScroll();
+            startNextScroll(response.getItems().length);
         } catch (Throwable t) {
             finishHim(t);
         }
     }
 
-    void startNextScroll() {
+    void startNextScroll(int lastBatchSize) {
```

Comment:
I know these aren't public methods, but javadocs would go a long way towards understanding whether this is the search batch size, or the bulk index response batch size (the later, from reading the code)